client/activities/activity-button.tsx

```diff
@@ -169,6 +169,25 @@ export const ActivityButton = React.memo(
         },
         [disabled, hotkey, onClick],
       )
+      const getLabelJsx = useMemo(() => {
```

Comment:
nit: `getLabelJsx` makes this seem like a function when it's really the actual value

maybe call this `labelElems` and use that instead of `labelJsx` inside the function as well?